wavefront_lambda/__init__.py

```diff
@@ -62,9 +74,21 @@ def wavefront_wrapper(*args, **kwargs):
         server = os.environ.get('WAVEFRONT_URL')
         if not server:
             raise ValueError("Environment variable WAVEFRONT_URL is not set.")
-        auth_token = os.environ.get('WAVEFRONT_API_TOKEN')
-        if not auth_token:
-            raise ValueError("Environment variable WAVEFRONT_API_TOKEN is not set.")
+        proxy = False
+        try:
+            int(os.getenv('WAVEFRONT_PROXY', "0"))
+        except ValueError:
+            pass
+        finally:
+            proxy = bool(proxy)
+        # WAVEFRONT_PORT PORT default 2878
+        port = 2878
+        if proxy:
+            port = os.environ.get("WAVEFRONT_PORT", port)
+        else:
+            auth_token = os.environ.get('WAVEFRONT_API_TOKEN')
```

Comment:
If someone uses the proxy, 'else' lines of code won't be executed and Interpreter would throw " 'auth_token' is not defined " on Line 122 .
This why you will need to define auth_token say  "auth_token = None" before the if-else lines of code at line86

Reply:
you are correct. I forget about this when moved auth_token inside the else.

Reply:
just pushed the fix